<div class="profile-request-error">
  <div class="orcid-request">
    <h3>We have the following request on record</h3>
    <dl>
      <dt>Given Names</dt>
      <dd><%= profile_request_in_error.given_names %></dd>
      <dt>Family Name</dt>
      <dd><%= profile_request_in_error.family_name %></dd>
      <dt>Primary Email</dt>
      <dd><%= profile_request_in_error.primary_email %></dd>
      <dt>Reqeusted At</dt>
      <dd><time datetime='<%= profile_request_in_error.created_at.in_time_zone %>'><%= time_ago_in_words(profile_request_in_error.created_at) %></time> ago.</dd>
      <dt>Requested By</dt>
      <dd><%= profile_request_in_error.user %></dd>
    </dl>
  </div>

  <div class="orcid-response">
    <h3>ORCID responded with</h3>
    <p class="error-message"><%= profile_request_in_error.error_message %></p>
  </div>

  <div class="orcid-actions">
    <%= link_to(
        t('orcid/profile_connection.profile_request_destroy', scope: 'helpers.label'),
        orcid.profile_request_path ,
        class: 'btn cancel-profile-request', method: :delete
      ) %>
  </div>
</div>
